tensorflow/core/debug/debug_grpc_io_utils_test.cc

```diff
@@ -25,6 +25,9 @@ limitations under the License.
 #include "tensorflow/core/platform/env.h"
 #include "tensorflow/core/util/event.pb.h"
 
+// TODO(cais): Support grpc:// debug URLs in open source once Python grpc
+//   genrule becomes available. See b/23796275.
+#if defined(PLATFORM_GOOGLE)
```

Comment:
Yeah - the concern about excluding it using tf_tests.cmake is Bazel OSS builds, because the non-test code in debug_io_util.cc are guarded with PLATFORM_GOOGLE. I guess it's better to submit this change from internal remove this test file from OSS altogether for now. WDYT?